### ISSUE TITLE
Add a way to suppress Call.debug output

### ISSUE THREAD

**Issue by gilgamec** [plredmond/GPipe-GLFW#33]
When running I get loads of debug output, such as
```
[0.164s, ThreadId 1 has Nothing]: attaching Just (Window {unWindow = 0x00000000031d6f00}), reason: contextDoAsync required a specific context
```
This debugging information seems to be output from several of the internal context functions, using the function ```Call.debug```.  There doesn't seem to be a way to suppress this output (besides suppressing all of stderr, at least).  Could you add, at the very least, a compile-time flag to get rid of this output?

**Comment by plredmond** [comment]
Yeah, that output is a result of this library still somewhat being an
experiment. Thanks for the heads up! I'll get rid of it soon.
